lib/connection.js

```diff
@@ -83,6 +88,7 @@ function getSocket(url, options) {
 
     socket.addEventListener('message', handleMessage);
     socket.addEventListener('close', closeMessage);
+    socket.addEventListener('error', closeMessage);
```

Comment:
We should also remove this on line 76 (`case 'auth_ok'`)

Reply:
Done